client/dashboard/sites/settings-site-visibility/privacy-form.tsx

```diff
@@ -97,6 +118,12 @@ export function PrivacyForm( {
 	settings: SiteSettings;
 	mutation: UseMutationResult< Partial< SiteSettings >, Error, Partial< SiteSettings >, unknown >;
 } ) {
+	const { data: domains = [] } = useQuery( siteDomainsQuery( site.ID ) );
```

Comment:
Nit: since the query is ensured in the loader, we can use a more type-safe way

```suggestion
	const { data: domains } = useSuspenseQuery( siteDomainsQuery( site.ID ) );
```